src/cider/nrepl/middleware/test.clj

```diff
@@ -67,10 +67,12 @@
 (defn- print-object
   "Print `object` using pprint or a custom print-method, if available."
   [object]
-  (let [print-fn (if (= (get-method print-method (:type (meta object)))
-                        (get-method print-method :default))
-                   pp/pprint
-                   println)]
+  (let [custom-type (:type (meta object))
```

Comment:
`(:type (meta object))` is clearly `nil` for any standard Clojure data type and `(get-method print-method nil)` is never `(get-method print-method :default)` (it is `(get-method print-method nil)`).

Reply:
Can you add a test that explicitly exercises this dispatch?